tools/serverpod_cli/lib/src/generator/dart/library_generators/entities_library_generator.dart

```diff
@@ -65,27 +68,30 @@ class SerializableEntityLibraryGenerator {
             _buildEntityImplicitClass(className, classDefinition),
         ]);
 
-        if (serverCode && tableName != null) {
+        if (serverCode && (tableName != null || viewName != null)) {
```

Comment:
Suggestion: Break out this check to a method with a fitting name and reference.

The check inside `(...)` is reflected in multiple places in the code, better to break it out so that it has a shared definition.